spec/views/documents/_timeline_entry.html.erb_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.describe "documents/_timeline_entry.html.erb" do
+  describe "timeline entry" do
+    it "shows a timeline entry without an author" do
+      timeline_entry = create(:timeline_entry,
+                              created_by: nil)
+      render partial: "documents/timeline_entry", locals: { entry: timeline_entry }
+      expect(rendered).not_to have_content(I18n.t!("documents.history.by"))
+      expect(rendered).to have_content(I18n.t!("documents.history.entry_types.created"))
+    end
+
+    it "shows a Whitehall timeline entry with an author" do
```

Comment:
Looks like a copy and paste issue here as this isn't a Whitehall timeline entry.